e2e/cypress/integration/api/iam/08_projects_api_nodemanager_update.ts

```diff
@@ -0,0 +1,288 @@
+import { describeIfIAMV2p1 } from '../../constants';
+import { uuidv4 } from '../../helpers';
+
+describeIfIAMV2p1('Nodemanager project update tagging', () => {
+  const cypressPrefix = 'test-nodemanager-update';
+
+
+  const projectsWithRule = [
+    // This test is commented out because there is a current limit of 6 projects allowed
```

Comment:
I am going to create a separate PR to increase the number of projects allowed and then uncomment all test with a seventh project.